src/IconButton/IconButton.test.jsx

```diff
@@ -13,21 +12,15 @@ describe('<IconButton />', () => {
     icon,
     variant,
   };
-  it('passes the icon and alt text to the icon', () => {
```

Comment:
We should probably still test that we're passing alt text to the button.

Reply:
Oh, so, as we speak I'm writing a test using `react-testing-library`, and I'm using `screen.getByAltText` to get the button. I'm kind of inclined *not* to change it to `aria-described` by, because that makes it actually kind of hard for `react-testing-library` to access it. I guess I can use `data-testid`, if we're passing all props along, but I'm not sure we are right now.